oeps/oep-0001.rst

```diff
@@ -0,0 +1,334 @@
+obsolete.
+
+The possible paths of the status of OEPs are as follows:
+
+.. image:: oep-0001/state-flow.png
+
+Please note that OEP statuses do not necessary coincide with the status of
+the pull request that contains the OEP. For example, OEPs that have been "Rejected"
+should still be merged, but should be marked as Rejected. This preserves the rationale
+and description of the OEP in the generated documentation.
+
+Likewise, an OEP that is in "Draft" may be merged to capture a set of edits,
+and to make the proposal more visible to community comment. From that point, additional
+pull requests can be opened to edit the "Draft" OEP, until it converges to being
+either "Accepted" or "Rejected".
+
+OEP Roles
+---------
+
+Each OEP will have an Arbiter (as described in `OEP Review & Resolution`_). The
+Arbiter will be chosen by the edX Chief Architect (currently Eddie Fagin). The Arbiter
+will be the person making the final decision on whether the OEP should be Accepted,
+and as such, they should be knowledgable about the contents of the proposal, and
+willing to listen to arguments both for and against it by the rest of the community.
+The Author of the OEP will never be selected as the Arbiter of that OEP.
+
+The Arbiter is also responsible for helping the Author to elicit feedback from
+the community on the OEP, and for helping to move the OEP towards a final decision
+(whether that decision is Accepted, Rejected or Deferred). The Arbiter (in
+discussion with the Author) can merge an in-progress OEP (if it has reached a
+stage of relative stability) to allow for additional incremental updates.
+
+Finally, the Arbiter is responsible for the decision to transfer an OEP if the
+original Author has become unresponsive (as described in `Transferring OEP Ownership`_).
+
+
+OEP Maintenance
+---------------
+
+In general, OEPs are no longer modified after they have reached the Final state.
+They can be replaced by subsequent OEPs, however (OEPs that have been replaced
+are given the status "Replaced").
+
+The choice of whether an edit to an OEP should be allowed or whether a new OEP
+should be published is up to the Arbiter of the original OEP. However, as a
+general guideline, the following would not require a replacement OEP::
+
+    * Formatting changes
+    * Grammatical and spelling corrections
+    * Adding links to additional relevant resources and discussions
+    * Additional diagrams or clarifying material (as long as the Arbiter agrees
+      that the substance of the OEP isn't changed).
+
+The following should be build as a replacement OEP::
+
+    * Changing the choice of technology in a Best Practice OEP (such as the
+      which test-runner should be used).
+    * Changing how a set of services is separated in an Architecture OEP (for
+      example, splitting one service into two, or combining two services into one).
+
+What belongs in a successful OEP?
+=================================
+Each OEP should have the following parts:
```

Comment:
I think this is a `should` vs `must` distinction, in my mind. Successful OEPs are likely to need to address all of these sections, but the process doesn't require it.